src/traces/scattergl/index.js

```diff
@@ -13,10 +13,12 @@ var ScatterGl = {};
 ScatterGl.attributes = require('./attributes');
 ScatterGl.supplyDefaults = require('./defaults');
 ScatterGl.colorbar = require('../scatter/colorbar');
+ScatterGl.hoverPoints = require('../scatter/hover');
 
 // reuse the Scatter3D 'dummy' calc step so that legends know what to do
-ScatterGl.calc = require('../scatter3d/calc');
+ScatterGl.calc = require('../scatter/calc');
```

Comment:
@dfcreative can you share benchmark script with me (e.g. via a GitHub gist or codePen)? I'm seeing some a significant slow down.